package.json

```diff
@@ -57,13 +53,13 @@
   "dependencies": {},
   "devDependencies": {
     "benchmark": "^2.1.0",
-    "browserify": "^13.0.0",
-    "browserify-istanbul": "^1.0.0",
-    "coveralls": "2.11.9",
-    "eslint": "^2.9.0",
-    "eslint-config-strict": "^8.5.0",
-    "eslint-plugin-filenames": "^0.2.0",
-    "ghooks": "^1.0.1",
+    "browserify": "^14.3.0",
```

Comment:
FYI, while Browserify `14` _technically_ no longer supports IE10, the main reason is due to it upgrading the `Buffer` module, which we don't make use of - so I feel like we can upgrade for this case - as it means less effort in trying to determine what packages need upgrading later on. However I also can see this is a slippery slope - as all we need to do is reference `Buffer` once to break browserify builds for IE10 - so I'm happy to take direction here.